x-pack/plugins/secops/public/store/local/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export * from './actions';
+export * from './reducer';
```

Comment:
This is better then my previous PR where I was manually pulling each export out of each package. I forgot about this. No changes, just a request to keep an eye out on my PR's to let me know to use this for `index.ts` on my PR's to reduce friction and code noise.